server/build/docker/keycloak/README.md

```diff
@@ -18,44 +18,12 @@ To use this keycloak image, we suggest you to use this configuration settings:
 - First Name Attribute: `firstName`
 - Last Name Attribute: `lastName`
 
-or overwrite your SamleSettings section with this settings in your config.json file (if you are not using
-database configuration) and restart the server:
+or overwrite your SamleSettings section in your config.json file by running `make config-saml` and restarting your server.
```

Comment:
Should `make config-saml` be the only way to configure the local dev server? i.e. do we even need to spell out the config value here?